addon/components/o-s-s/modal.js

```diff
@@ -18,6 +19,10 @@ export default class OssModalComponent extends Component {
     return { ...DEFAULT_OPTIONS, ...(this.args.options || {}) };
   }
 
+  get container() {
+    return isTesting() ? null : this.options.container;
```

Comment:
I think we tried but I can't remember the result. What happens when running tests if we set the 'ember-testing-container' instead of null here ?

Reply:
It works, but if I set a random value, it works too. It's weird...

Reply:
the backdrop was still in the body, also